pkg/kritis/cryptolib/pgp_test.go

```diff
@@ -136,3 +304,100 @@ func TestVerifyPgp(t *testing.T) {
 		})
 	}
 }
+
+func TestCreateAttestation_WellFormedAttestation(t *testing.T) {
+	tcs := []struct {
+		name               string
+		privateKey         []byte
+		expectNewSignerErr bool
+	}{
+		{
+			name:               "correct signature over payload",
+			privateKey:         []byte(gpgPrivateKey),
+			expectNewSignerErr: false,
+		},
+		{
+			name:               "invalid private key",
+			privateKey:         []byte("invalid-private-key"),
+			expectNewSignerErr: true,
```

Comment:
Separate out TestCreatePgpSigner. These two test cases are not reusing any logic

Reply:
Done.